Makefile

```diff
@@ -1,6 +1,9 @@
 .SILENT: ;
 
-test: test-phpunit test-phpspec
+test: cs-check test-phpunit test-phpspec
+
+cs-check:
+	vendor/bin/php-cs-fixer fix --dry-run --stop-on-violation
```

Comment:
May I suggest the following instead?

```suggestion
	vendor/bin/php-cs-fixer fix --diff --diff-format=udiff --dry-run
```

This will prevent it stopping on the first violation and it will print out a diff of each violation.

Running this against another project of mine, looks like the following:

```diff
$ php-cs-fixer fix --diff --diff-format=udiff --dry-run ./src
Using cache file ".php_cs.cache".
Paths from configuration file have been overridden by paths provided as command arguments.
   1) src/Migrations/Version20200330175548.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -12,23 +12,23 @@
  */
 final class Version20200330175548 extends AbstractMigration
 {
-    public function getDescription() : string
+    public function getDescription(): string
     {
         return 'Drop completed_at column from employer_user_invitations';
     }
 
-    public function up(Schema $schema) : void
+    public function up(Schema $schema): void
     {
         // this up() migration is auto-generated, please modify it to your needs
-        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
 
         $this->addSql('ALTER TABLE employer_user_invitations DROP completed_at');
     }
 
-    public function down(Schema $schema) : void
+    public function down(Schema $schema): void
     {
         // this down() migration is auto-generated, please modify it to your needs
-        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
 
         $this->addSql('ALTER TABLE employer_user_invitations ADD completed_at TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL');
     }

      ----------- end diff -----------

   2) src/Migrations/Version20200330183847.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -12,23 +12,23 @@
  */
 final class Version20200330183847 extends AbstractMigration
 {
-    public function getDescription() : string
+    public function getDescription(): string
     {
         return 'Drop multi-column index for columns employer_id and email on employer_user_invitations';
     }
 
-    public function up(Schema $schema) : void
+    public function up(Schema $schema): void
     {
         // this up() migration is auto-generated, please modify it to your needs
-        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
 
         $this->addSql('DROP INDEX uniq_945ab10241cd9e7ae7927c74');
     }
 
-    public function down(Schema $schema) : void
+    public function down(Schema $schema): void
     {
         // this down() migration is auto-generated, please modify it to your needs
-        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
 
         $this->addSql('CREATE UNIQUE INDEX uniq_945ab10241cd9e7ae7927c74 ON employer_user_invitations (employer_id, email)');
     }

      ----------- end diff -----------

   3) src/Services/EmailChangeRequestManager.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -45,7 +45,7 @@
             $this->em->flush();
         } catch (\Throwable $th) {
             $this->logger->error('Uneble to create new email change request: failed to remove pending email change requests', [
-                'user'=> $user->getId(),
+                'user' => $user->getId(),
             ]);
             throw $th;
         }
@@ -62,7 +62,7 @@
             }
         } catch (\Throwable $th) {
             $this->logger->error('Uneble to create new email change request: failed to remove existing employer invites', [
-                'user'=> $user->getId(),
+                'user' => $user->getId(),
                 'new_email' => $newEmail,
                 'employer_user_invitation' => $existingEmployerInvite->getId(),
             ]);
@@ -93,10 +93,9 @@
                 'bodyHtml' => $this->templating->render('emails/verify-email-change-request.html.twig', $emailParams),
                 'bodyText' => $this->templating->render('emails/verify-email-change-request.txt.twig', $emailParams),
             ]);
-
         } catch (\Throwable $th) {
             $this->logger->error('Uneble to create new email change request', [
-                'user'=> $user->getId(),
+                'user' => $user->getId(),
                 'new_email' => $newEmail,
             ]);
             throw $th;

      ----------- end diff -----------

   4) src/Services/EmployerManager.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -20,6 +20,7 @@
         $this->twig = $twig;
         $this->queueManager = $queueManager;
     }
+
     /**
      * @throws \InvalidArgumentException
      */
@@ -44,6 +45,5 @@
             'bodyHtml' => $this->twig->render('emails/employer-invite.html.twig', ['invitation' => $invitation]),
             'bodyText' => $this->twig->render('emails/employer-invite.txt.twig', ['invitation' => $invitation]),
         ]);
-
     }
 }

      ----------- end diff -----------

   5) src/Repository/UserRepository.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -94,7 +94,7 @@
     {
         $conn = $this->getEntityManager()->getConnection();
 
-        $sql = "DELETE FROM sessions WHERE sess_data LIKE :email";
+        $sql = 'DELETE FROM sessions WHERE sess_data LIKE :email';
         $stmt = $conn->prepare($sql);
         $stmt->execute(['email' => sprintf('%%%s%%', $user->getEmail())]);
     }

      ----------- end diff -----------

   6) src/Controller/Api/Employer/UsersController.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -7,8 +7,8 @@
 use App\Entity\EmployerUserInvitation;
 use App\Entity\User;
 use App\Enum\EmployerUserState;
+use App\Form\Employer\InviteEmployerUserFormType;
 use App\Form\Employer\UpdateEmployerUserFormType;
-use App\Form\Employer\InviteEmployerUserFormType;
 use App\Repository\EmployerUserInvitationRepository;
 use App\Repository\UserRepository;
 use App\Services\EmailChangeRequestManager;

      ----------- end diff -----------


Checked all files in 0.128 seconds, 14.000 MB memory used
```

Reply:
Oh! I like that better! I'll change that now.